python/ray/autoscaler/_private/util.py

```diff
@@ -724,6 +724,18 @@ def filter_placement_group_from_bundle(bundle: ResourceBundle):
     return demand_lines
 
 
+def get_constraints_report(lm_summary: LoadMetricsSummary):
```

Comment:
nit: just make the naming style consistent with `get_demand_report`.

```suggestion
def get_constraint_report(lm_summary: LoadMetricsSummary):
```